src/virtual_secondary/partialverificationsecondary.h

```diff
@@ -44,8 +44,8 @@ class PartialVerificationSecondary : public SecondaryInterface {
   int getRootVersion(bool director) const override;
   bool putRoot(const std::string& root, bool director) override;
 
-  bool sendFirmware(const std::string& data) override;
-  data::ResultCode::Numeric install(const std::string& target_name) override;
+  // bool sendFirmware(const std::string& data) override;
```

Comment:
?

Reply:
removed

Reply:
> Why was it necessary to move this, though?

Because MultiPartHasher depends on Uptane::Hash now and I didn't want to make crypto.h depend on uptane/tuf.h since the later depends on crypto.h so we get a circle dependency here which can be overcomed. But, I think a notion of hash is not Uptane specific stuff, so I thought we can move Uptane::Hash just to Hash?

Reply:
> But, I think a notion of hash is not Uptane specific stuff, so I thought we can move Uptane::Hash just to Hash?

Agreed, the hashing logic should be removed to either the crypto section or its own new file.

Reply:
So, can I get rid of this Uptane namespace from the Hash declaration? 

>  the hashing logic should be removed to either the crypto section or its own new file.
What is the crypto section?

Reply:
> So, can I get rid of this Uptane namespace from the Hash declaration?

Yeah, I don't think it needs to be in the Uptane namespace.

> What is the crypto section?

I just meant "crypto section of the code". I should've just said "file". :)

Reply:
So, I am moving Hash out of the Uptane namespace and leave its implementation in crypto.h/cc files?